src/Authentik/composer.json

```diff
@@ -0,0 +1,19 @@
+{
+    "name": "socialiteproviders/authentik",
+    "description": "Authentik OAuth2 Provider for Laravel Socialite",
+    "license": "MIT",
```

Comment:
Can you also add the missing keywords section?

Reply:
Also the support section is missing.

Reply:
Added